qemu/tests/virtio_scsi_mq.py

```diff
@@ -177,6 +176,9 @@ def proc_interrupts_results(results):
         raise error.TestError("Disks are not all show up in system. Output "
                               "from the check command: %s" % output)
     fill_cmd = fill_cmd.rstrip("&&")
+    # As Bug 1177332 exists, mq is not supported completely.
+    # So don't considering performance currently, dd_timeout is longer.
+    dd_timeout = dd_timeout * (images_num * 2)
```

Comment:
any difference between   a * b *c and a * (b * c) ? it looks there no need to add  '()'.

Thanks,
Xu